Common/WALinuxAgent-2.0.16/waagent

```diff
@@ -6535,6 +6538,27 @@ def dist_info_SLES15():
     return ["","",""]
 
 
+def dist_info_opensuse15():
+    os_release_filepath = "/etc/os-release"
+    if not os.path.isfile(os_release_filepath):
+        return ["","",""]
+    info = open(os_release_filepath).readlines()
```

Comment:
OS release is readable for everyone in every distro. Yes, it could throw if somebody were to lock down something as fundamental as /etc/os-release but that would be a really far-fetched corner case.